frontend/src/container/SideNav/SideNav.tsx

```diff
@@ -259,6 +255,42 @@ function SideNav({
 		? ROUTES.ORG_SETTINGS
 		: ROUTES.SETTINGS;
 
+	useEffect(() => {
+		registerShortcut(GlobalShortcuts.SidebarCollapse, onCollapse);
```

Comment:
Rather than manually adding registerShortcut every time we add any new GlobalShortcuts, can we create a map of GlobalShortcuts and Routes 

E.g: 

```typescript
type GlobalShortcutsType = typeof GlobalShortcuts;
type ShortcutsMap = {
  [K in keyof GlobalShortcutsType]: K extends keyof GlobalShortcutsType ? string | null : never;
};

const shortcutsMap: ShortcutsMap = {
  [GlobalShortcuts.SidebarCollapse]: null,
  [GlobalShortcuts.NavigateToServices]: ROUTES.APPLICATION,
  [GlobalShortcuts.NavigateToTraces]: ROUTES.TRACE,
  [GlobalShortcuts.NavigateToLogs]: ROUTES.LOGS,
  [GlobalShortcuts.NavigateToDashboards]: ROUTES.ALL_DASHBOARD,
  [GlobalShortcuts.NavigateToAlerts]: ROUTES.LIST_ALL_ALERT,
  [GlobalShortcuts.NavigateToExceptions]: ROUTES.ALL_ERROR,
};
```


Can then inside use effect

```typescript
useEffect(() => {
  const shortcuts = Object.entries(shortcutsMap);

  shortcuts.forEach(([shortcut, route]) => {
    if (route) {
      registerShortcut(shortcut, () => onClickHandler(route));
    } else {
      registerShortcut(shortcut, onCollapse);
    }
  });

  return () => {
    shortcuts.forEach(([shortcut]) => {
      deregisterShortcut(shortcut);
    });
  };
}, [deregisterShortcut, onClickHandler, onCollapse, registerShortcut]);
```

Reply:
will eventually add a group register method to the `useKeyboardHotkeys` to ease out the pain of multiple registers